core/src/test/scala/vegas/fixtures/VegasPlots.scala

```diff
@@ -29,7 +29,15 @@ object VegasPlots {
           encodeY2("people", aggregate=Q3)
       )
 
-  val plots: List[SpecBuilder] = ValuePlot :: IQRPlot :: Nil
+  val BinPlot =
+    Vegas("Plot with binning").
+      withDataURL(Movies).
+      mark(Point).
+      encodeY("IMDB_Rating", Quantitative, bin=Bin(maxbins=10.0)).
+      encodeX("Rotten_Tomatoes_Rating", Quantitative, bin=Bin(maxbins = Some(10.0))).
```

Comment:
My second commit (before creating the PR) has this fix. Not sure, why the PR diff didn't pick it up. 

I created this from the link you had in the issue. I'll be moving this to BasicPlots now since the json is present in example-specs.